csi.proto

```diff
@@ -884,6 +894,10 @@ message ControllerServiceCapability {
 
       // See VolumeExpansion for details.
       EXPAND_VOLUME = 9;
+
+      // Indicates the SP supports the
+      // ListVolumesResponse.entry.published_nodes field
+      LIST_VOLUMES_PUBLISHED_NODES = 10;
```

Comment:
Did we want a capability for this one? Seems like a unnecessary knob.

Reply:
must be optional for backwards compatibility of the spec. Otherwise anyone implementing `ListVolumes` will be forced to also implement `PUBLISHED_NODES`.

Callers can then tell the difference between "no published nodes because capability not supported" and "no published nodes because the volume is not published on any nodes"